docs/en/Storage.md

```diff
@@ -1,132 +1,261 @@
-# Storage
+# Storage Types
 
-The platform has a range of different types of storage meant for a variety of
-use-cases, so this storage section applies whether you're experimenting,
-creating pipelines, or publishing.
+The platform provides two main types of storage:
 
-At the surface, there are two kinds of storage
+- Disk (also called Volumes on the Notebook Server creation screen)
+- Bucket ("Blob" or S3 storage, provided through MinIO)
 
-- Disks (also called Volumes)
-- Buckets (S3 or "Blob" storage)
+Depending on your use case, either may be most suitable:
+
+|   Type |                                                                                                         Simultaneous Users |                                                   Speed | Total size               |
+| -----: | -------------------------------------------------------------------------------------------------------------------------: | ------------------------------------------------------: | ------------------------ |
+|   Disk |                                       One compute instance at a time (one notebook server, step in kubeflow pipeline, ...) |                        Fastest (throughput and latency) | <=512GB total per drive  |
+| Bucket | Many compute instances at once (ex: multiple training processes on different notebook servers all accessing the same data) | Fast-ish (Fast download, modest upload, modest latency) | Infinite (within reason) |
```

Comment:
As above, just simplify the language a bit. Like "Simultaneous access from any number of servers"